aiotg/bot.py

```diff
@@ -519,6 +521,11 @@ def __del__(self):
 
     def _process_message(self, message):
         chat = Chat.from_message(self, message)
+        conversation = self._conversations.get(chat.id)
+
+        if conversation:
+            conversation.put_nowait(message)
+            return
```

Comment:
Not sure about return here. I understand that we probably want to skip normal processing for conversation replies, but bluntly returning here doesn't seem to be right 🤔